src/java/us/kbase/common/utils/NetUtils.java

```diff
@@ -26,7 +26,8 @@
         List<String> ret = new ArrayList<String>();
         for (Enumeration<NetworkInterface> en = NetworkInterface.getNetworkInterfaces(); en.hasMoreElements();) {
             NetworkInterface intf = en.nextElement();
-            if (!intf.isUp()) continue;
+            // breaks linux, if required for windows needs to be os-dependent
```

Comment:
Sorry... I think it is fine with Windows with the current code... 
My earlier code has to use this...